keep-ui/app/rules/CorrelationSidebar/AlertsFoundBadge.tsx

```diff
@@ -40,8 +42,9 @@ export const AlertsFoundBadge = ({
             src={`/icons/${source}-icon.png`}
           />
         ))}
+        {vertical && <span className="basis-full"></span>}
         <span className="ml-4">
-          {alertsFound.length} alert(s) were found matching this condition
+          {alertsFound.length} alert(s) were found{vertical && <br/>}matching this condition
```

Comment:
So - for the "specific group" it makes sense for me to say "1000 alerts were found", but for the whole rule (x groups) - what does it mean "1000 alerts were found matching this condition"? its not a condition, and its not the outcome I would think about. I would think "This rule correlate 1500 alerts into 5 incidents" which is WAY more valuable.

(also - can we write alert if alerts == 1 else alerts?)

Reply:
But we don't actually correlate historical data. It can be confusing

Reply:
I Will do it in the next iteration - it will require quite a bit of work at the backend

Reply:
https://github.com/keephq/keep/issues/1737